### PR TITLE
OC3-26 [TASK] added option to enable company view mode

### PR DESCRIPTION
**Pull request by bassurman**

### CODE REVIEW

File: system/library/helperbm.php [hunk 1]
```diff
@@ -28,6 +28,10 @@ class Helperbm {
         'country'  => 'se'
     ];
 
+    protected $companyViewMap = [
```
Comment (MattiasNilsson) [inline system/library/helperbm.php:31]: This is unnecessary to have it.

File: system/library/helperbm.php [hunk 2]
```diff
@@ -148,6 +152,14 @@ public function getInvoiceFeeTax()
         return  $this->config->get('payment_billmate_checkout_inv_fee_tax');
     }
 
+    /**
+     * @return string
```
Comment (MattiasNilsson) [inline system/library/helperbm.php:156]: It return a bool.

File: system/library/helperbm.php [hunk 3]
```diff
@@ -148,6 +152,14 @@ public function getInvoiceFeeTax()
...
+     */
+    public function isCompanyView()
+    {
+        return  (bool)$this->config->get('payment_billmate_checkout_is_company_view');
```
Comment (MattiasNilsson) [inline system/library/helperbm.php:160]: Should be: 

`return (bool)$this->config->get('payment_billmate_checkout_is_company_view');`

Comment (bassurman) [reply system/library/helperbm.php:160]: Are you sure that it is the best way to write this row? Which doc describes that we need to write it in this way?

Comment (bassurman) [reply system/library/helperbm.php:160]: Oh, I got, it is about two white spaces.

**Pull request merged**